libs/ECS/include/Scene.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by Daan van Donk on 10/05/2022.
+//
+
+#ifndef PROFTAAK24_SCENE_H
+#define PROFTAAK24_SCENE_H
+
+
+#include <vector>
+#include "GameObject.h"
+
+class Scene
+{
+public:
+    std::vector<GameObject> gameobjects;
+    float deltaTime;
```

Comment:
deltaTime wordt nog nergens gebruikt. Ik gok dat er in SceneManager::UpdatePoll() een nieuwe deltaTime moet worden berekend.
Daarnaast is deltaTime ook public, dus game objects kunnen ook deltaTime aanpassen. Misschien een getter/setter omheen zetten op een manier zodat alleen de SceneManager deze deltaTime kan aanpassen